### PR TITLE
feat: Implement add a new external visio connector - EXO-68227_68478

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

File: webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue [hunk 1]
```diff
@@ -82,6 +74,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
             <v-spacer />
             <v-switch      
               v-model="isUserConnector"
+              :aria-label="$t('externalVisio.tooltip.userModuleConnector', {0: isUserConnector ? 'Disable' : 'Enable'})"
```
Comment (azayati) [inline webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue]: 'Disable' and 'Enable' labels will not be translated like that no ?

File: webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue [hunk 2]
```diff
@@ -94,6 +87,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
             <v-spacer />
             <v-switch      
               v-model="isSpaceConnector"
+              :aria-label="$t('externalVisio.tooltip.spaceModuleConnector', {0: isSpaceConnector ? 'Disable' : 'Enable'})"
```
Comment (azayati) [inline webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue]: 'Disable' and 'Enable' labels will not be translated like that no ?

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue (hunk 1 above)
Comment (IlhemEssaadi) [reply webapp/src/main/webapp/vue-app/external-visio/components/ExternalVisioDrawer.vue]: > 'Disable' and 'Enable' labels will not be translated like that no ?

Yes, the 'Disable' and 'Enable' labels will not be translated, this issue has been fixed.

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**